pom.xml

```diff
@@ -89,13 +89,14 @@
         <jaxws-version>2.3.1</jaxws-version>
 
         <!-- Logging & commons (aligned) -->
-        <log4j-version>2.19.0</log4j-version>
+        <log4j.version>2.19.0</log4j.version>
+        <slf4j.version>1.7.25</slf4j.version>
         <commons-logging-version>1.2</commons-logging-version>
-        <commons-lang-version>2.3</commons-lang-version>
+        <commons-lang-version>3.17.0</commons-lang-version>
         <commons-collections-version>3.2.2</commons-collections-version>
         <commons-beanutils-version>1.9.4</commons-beanutils-version>
         <commons-dbcp-version>1.2.2</commons-dbcp-version>
-        <commons-codec-version>1.4</commons-codec-version>
+        <commons-codec-version>1.16.0</commons-codec-version>
```

Comment:
Can you please briefly report this in PR description with a more coherent PR title? Thanks.